File: Python/Exercicio4_calculos_lista.py
# Dada uma lista de números, calcule:
#   - a soma de todos os números
#   - a média de todos os números
#   - O maior número
#   - o menor número 
numeros = [-2,-5,-3,-1]
soma =0
media =0
maiorNumero = numeros[0]
menorNumero =0
for indice, item in enumerate(numeros):
    if(maiorNumero<item):
        maiorNumero = item
    soma = soma+item
menorNumero = maiorNumero
for indice, item in enumerate(numeros):
    if(menorNumero>item):
        menorNumero = item
media = soma/len(numeros)
print(f'O menor número é { menorNumero}\nO maior número é {maiorNumero}\nA soma dos números é {soma} \nA media é {media}')

File: C/Exercicio4_calculos_lista.c
/*
# Dada uma lista de números, calcule:
#   - a soma de todos os números
#   - a média de todos os números
#   - O maior número
#   - o menor número
*/

#include <stdio.h>
#include <stdlib.h>
#define TAM 4
int main()
{
    int vet[TAM]        = {-2,-3,100,-5};
    int MaiorElmento  = vet[0];
    int MenorElemento = vet[0];
    int soma          =  0;
    float media       =  0.0;

    for(int i=0; i<TAM; i++)
    {
        soma = soma + vet[i];
        if(MenorElemento>vet[i])
            MenorElemento = vet[i];


    }//EndFor_MenorElemento

    for(int i=0;i<TAM;i++)
    {
        if(MaiorElmento<vet[i])
            MaiorElmento = vet[i];
    }
    media = (float)soma/TAM;
    printf("Menor elemento: %d\nMaior elemento: %d\nSoma: %d\nMedia: %.2f ",
                MenorElemento,MaiorElmento, soma, media);
    return 0;
}
